internal/converter/embdemd_openapi_converter.go

```diff
@@ -61,6 +63,8 @@ type EmbedMDToOpenAPIConverter interface {
 	// goverter:map Attributes ArtifactType | MapEmbedMDArtifactTypeModelArtifact
 	// goverter:map Attributes CreateTimeSinceEpoch | MapEmbedMDCreateTimeSinceEpochModelArtifact
 	// goverter:map Attributes LastUpdateTimeSinceEpoch | MapEmbedMDLastUpdateTimeSinceEpochModelArtifact
+	// goverter:map . ExperimentId | MapEmbedMDExperimentId
```

Comment:
```suggestion
	// goverter:map CustomProperties ExperimentId | MapEmbedMDExperimentId
```
This will allow the func to take `CustomProperties` param directly.